scripts/pi-hole/js/index.js

```diff
@@ -386,7 +386,7 @@ function updateClientsOverTime() {
         for (key in data.clients)
         {
             if (!{}.hasOwnProperty.call(data.clients, key)) continue;
-            labels.push(data.clients[key]);
+            labels.push(data.clients[key].name);
```

Comment:
We need to add a fallback option for when there is no host name for a given IP address